operators/src/engine/operator.rs

```diff
@@ -95,6 +95,9 @@ where
     fn execution_context(&self) -> &ExecutionContext {
         self.as_ref().execution_context()
     }
+    fn result_descriptor(&self) -> Self::Descriptor {
+        todo!()
```

Comment:
This is still to return

Reply:
9a58f7c73f62a9cbe95eb5289329fb09e730aee8